modules/core/test/v2/unit/coins/sol.ts

```diff
@@ -78,4 +83,20 @@ describe('SOL:', function () {
       await should.throws(() => basecoin.signTransaction('placeholder'), 'signTransaction method not implemented');
     });
   });
+
+  describe('Sign message', () => {
+    it('should sign message', async function () {
+      const signed = await basecoin.signMessage(
+        keypair,
+        'signed message',
+      );
+      signed.toString('base64').should.equal('s+7d/8aW/twfM/0wLSKOGxd9+LhDIiz/g0FfJ39ylJhQIkjK0RYPm/Y+gdeJ5DIy6K6h6gCXXESDomlv12DBBQ==');
+    });
```

Comment:
let's use a real solana transaction as the message. then verify the signed message can be deserialized using our transaction builder w/ expected payload.

Reply:
i dont undestand why i need to use a transaction, maybe change String message ? , this function is for sign a message is not for signed transaction

Reply:
sorry, there was some confusion on my end. this is okay